src/store/commands/types.js

```diff
@@ -5,4 +5,10 @@ const LOGIN = {
 };
 const LOGOUT = { SUCCESS: 'LOGOUT_SUCCESS' };
 
-export { LOGIN, LOGOUT };
+const GET_INTERNSHIPS = {
+  REQUEST: 'GET_INTERNSHIPS_REQUEST',
+  SUCCESS: 'GET_INTERNSHIPS_SUCCESS',
+  FAILURE: 'GET_INTERNSHIPS_FAILURE',
+};
+
```

Comment:
Please delete GET_INTERNSHIPS from types.js. You already have GET_INTERNSHIPS in action file